src/screens/Connectors/ConnectorUIUtils/PaymentMethod.res

```diff
@@ -165,7 +165,7 @@ module CardRenderer = {
                   <CheckBoxIcon isSelected={isSelected(value)} />
                 </div>
               </AddDataAttributes>
-              <p className=p2RegularTextStyle>
+              <p className=p2RegularTextStyle onClick={_ => removeOrAddMethods(value)}>
```

Comment:
Rather than adding the click functionality separate for both checkbox and text , can we add it to the parent div of both ?

Reply:
Sure @Riddhiagrawal001 , made the change. Thanks for suggesting!